src/palace/manager/api/axis.py

```diff
@@ -549,21 +564,32 @@ def patron_activity(
         pin: str | None,
         identifier: Identifier | None = None,
         internal_format: str | None = None,
+        debug_info: list[str] | None = None,
     ) -> list[AxisLoanInfo | HoldInfo]:
         if identifier:
             assert identifier.identifier is not None
             title_ids = [identifier.identifier]
         else:
             title_ids = None
+
         availability = self.availability(
             patron_id=patron.authorization_identifier, title_ids=title_ids
         )
-        return list(
+
+        availability_content_str = availability.content.decode("utf-8")
```

Comment:
Minor: Instead of decoding this yourself and assuming the character encoding is utf-8, you can do `availability.text` https://docs.python-requests.org/en/latest/api/#requests.Response.text which decodes it according to the request headers.